src/main/java/org/hmhb/user/UserController.java

```diff
@@ -59,6 +63,38 @@ public UserController(
     }
 
     /**
+     * Retrieves all {@link HmhbUser}s in the system and exports them to a
+     * CSV file.
+     *
+     * @param jwtToken the JWT auth token (needed in a query param because this
+     *                 method will be called via $window.open).
+     * @param response the {@link HttpServletResponse} to send the results to
+     */
+    @Timed
+    @RequestMapping(
+            method = RequestMethod.GET,
+            path = "/api/users/csv"
+    )
+    public void getAllAsCsv(
+            @RequestParam("token") String jwtToken,
```

Comment:
There isn't a way for me to set the auth header on the window.open, so we have to pass the token in as a query param.

````
$window.open('http://localhost:8080/api/users/csv?token=' + authenticationService.getToken(), '_blank');
````